### DISCUSSION TITLE
[Feature request] Settings to manipulate the horizontal tabs with the mouse

### DISCUSSION THREAD

**Discussion by Terkyz** [Floorp-Projects/Floorp#748] [Feature Request]
With [Sidebery](https://addons.mozilla.org/es/firefox/addon/sidebery/), you can configure the horizontal tab bar to use left-click to delete a tab when the mouse is over a tab, or use the scroll wheel to change between tabs. I think that something like this would be super awesome since, at least for me, it's pretty comfortable to use.

**Comment by CutterKnife** [comment]
This setting already exists. Check settings such as "Switch tabs by scrolling with your mouse" and "Close tabs with a double click".
